sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/healthcare/FhirIO.java

```diff
@@ -220,6 +230,26 @@ public static Import importResources(
     return new Import(fhirStore, tempDir, deadLetterDir, contentStructure);
   }
 
+  /**
+   * Export resources to GCS. Intended for use on non-empty FHIR stores
+   *
+   * @return the export
+   * @see ExportGcs
+   */
+  public static ExportGcs exportResourcesToGcs(ExportGcs.Options exportOptions) {
```

Comment:
It's a little unorthodox to pass a configuration object to a PTransform rather than configure it directly via `withXYZ` calls. 
Why did you choose to define ExportGcs.Options rather than make the ExportGcs class hold its own configuration?

I can think of one benefit - It seems that you can define multiple ExportGCS.Options and pass them down a PTransform to execute multiple exports. Did you have this in mind?

Reply:
Export FHIR resources doesn't have many options like import so I moved those config into ExportGcs class.